frontend/src/metabase/ui/components/overlays/Tooltip/Tooltip.styled.tsx

```diff
@@ -25,5 +25,16 @@ export const getTooltipOverrides = (): MantineThemeOverride["components"] => ({
         padding: "0.6rem 0.75rem",
       },
     }),
+    variants: {
+      multiline: _theme => {
+        return {
+          tooltip: {
+            whiteSpace: "normal",
+            maxWidth: "20rem",
+            overflowWrap: "break-word",
+          },
+        };
+      },
+    },
```

Comment:
I don't think we need a variant here, the mantine `maw` and `multiline` props should be sufficient

https://metaboat.slack.com/archives/C057WD5L0JG/p1715193241304909

Reply:
Yup, meant to delete this here